garage/tf/algos/npo.py

```diff
@@ -33,14 +33,33 @@ class NPO(BatchPolopt):
     """
     Natural Policy Gradient Optimization.
 
-    Attributes:
-        name(str): The name of the algorithm.
+    Args:
+        pg_loss(enum): The type of loss functions to use.
         lr_clip_range(float): The limit on the likelihood ratio between
             policies, as in PPO.
         max_kl_step(float): The maximum KL divergence between old and new
             policies, as in TRPO.
+        optimizer(object): The optimizer of the algorithm.
```

Comment:
is there an optimizer base class we can put here instead?

Reply:
No I don't think there is one. We don't have optimizer base class in garage/tf.

Reply:
Ok it's fine. we can just say (object) and note the optimizer package in the description